en-ar/jw300-tedtalk-baseline/notebook.ipynb

```diff
@@ -0,0 +1,1370 @@
+{
```

Comment:
<p>So, we should be using the global test sets, to ensure reproducability and we can use this result to compare to when later doing transfer learning. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/masakhane-io/masakhane/pull/59/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>so the problem with Arabic at first was that there was no test files for english and arabic in the repo, after consulting Julia i took the file test-any.en and its corresponding sentences in Arabic as the test files and then went along with the training. </p><p>this approach reported 0.0 test bleu score for the files, after going back to the test files that i made i found they are terrible in terms of alignment and even after preprocessing the text. so me and Amr (team-mate) figured we should take the same sentences lenght of the test-en.any file from the combined and shuffled datasets since we are reporting the baseline for both and this will aslo fix the alingment issue since we made sure about all sentences to be aligned for the combined dataframe </p>

Reply:
resolved, added explaination in the readme.md